tardis/apps/publication_workflow/components/Stepper/ProgressBar.jsx

```diff
@@ -0,0 +1,73 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled from '@emotion/styled';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/mytardis/mytardis/pullRequest?prid=7933947)
```suggestion
import styled from "@emotion/styled";
```